Platform/de.lproj/Localizable.strings

```diff
@@ -952,7 +952,7 @@
 "USB sharing not supported in this build of UTM." = "Die USB-Freigabe wird in dieser Version von UTM nicht unterstützt.";
 
 /* No comment provided by engineer. */
-"Use Command+Option (⌘+⌥) for input capture/release" = "Drücken Sie cmd+alt (⌘+⌥) um den Mauszeiger zu fangen oder freizugeben.";
+"Use Command+Option (⌘+⌥) for input capture/release" = "Drücken Sie CMD+OPT (⌘+⌥) um den Mauszeiger zu fangen oder freizugeben.";
```

Comment:
⚠️ 〽️ _v.s._
```suggestion
"Use Command+Option (⌘+⌥) for input capture/release" = "Drücken Sie CMD+OPT (⌘+⌥) um den Mauszeiger ausschließlich in UTM zu nutzen oder wieder für alle anderen Anwendungen freizugeben.";
```

Reply:
```suggestion
"Use Command+Option (⌘+⌥) for input capture/release" = "Drücken Sie CMD+OPT (⌘+⌥), um die Tastatur und den Mauszeiger ausschließlich in der VM zu nutzen oder wieder für alle anderen Anwendungen freizugeben.";
```
Consistency with below changes for capture/release.